templates/visuals/.api/v1.13.0/schema.capabilities.json

```diff
@@ -40,6 +40,10 @@
             "type": "boolean",
             "description": "Tells the host to include highlight data"
         },
+        "supportsSynchronizingFilterState": {
+            "type": "boolean",
+            "description": "Indicates whether synchronize slicer (vissual) across report pages"
```

Comment:
Please change to "Indicates whether the visual supports synchronization across report pages (for slicer visuals only)"